src/pydex_app/utils.py

```diff
@@ -88,3 +89,31 @@ def setup_logger(
     logger.setLevel(level)
 
     return logger
+
+
+def sign_order(order_hash, web3_instance, private_key):  # pylint: disable=redefined-outer-name
+    """Function to sign order_hash via eth-sign and convert ec_signature to
+    0x compatible signature
+    """
+
+    def to_32byte_hex(val):
+        """Method to convert value to bytes32 hex
+        """
```

Comment:
as per [PEP8 docstring convention](https://www.python.org/dev/peps/pep-0257/) for single line docstring docs could you close the comment on the same line? It is not a huge deal but preferred.